saleor/graphql/account/filters.py

```diff
@@ -0,0 +1,109 @@
+import django_filters
+from django.db.models import Count, Sum
+
+from saleor.graphql.utils import filter_by_query_param
+
+from ...account.models import User
+from ..core.filters import EnumFilter, ObjectTypeFilter
+from ..core.types.common import DateRangeInput, IntRangeInput, PriceRangeInput
+from .enums import StaffMemberStatus
+
+
+def filter_date_joined(qs, _, value):
+    from_date = value.get('from_date')
+    to_date = value.get('to_date')
+    if from_date:
+        qs = qs.filter(date_joined__date__gte=from_date)
+    if to_date:
+        qs = qs.filter(date_joined__date__lte=to_date)
+    return qs
+
+
+def filter_money_spent(qs, _, value):
+    qs = qs.annotate(money_spent=Sum('orders__total_gross'))
+    money_spent_lte = value.get('lte')
+    money_spent_gte = value.get('gte')
+    if money_spent_lte:
+        qs = qs.filter(money_spent__lte=money_spent_lte)
+    if money_spent_gte:
+        qs = qs.filter(money_spent__gte=money_spent_gte)
+    return qs
+
+
+def filter_number_of_orders(qs, _, value):
+    qs = qs.annotate(total_orders=Count('orders'))
+    gte = value.get('gte')
+    lte = value.get('lte')
```

Comment:
I believe it's a bit faster for python to do
```python
gte, lte = value.get('gte'), value.get('lte')
```

But I'm not 100% sure. Let me know what you think.

Reply:
I don't think it's faster but may be more readable if the two are related.